Jenkinsfile

```diff
@@ -2,6 +2,13 @@
 
 @Library('xmos_jenkins_shared_library@v0.39.0') _
 
+def clone_test_deps() {
+  dir("${WORKSPACE}") {
+    sh "git clone git@github.com:xmos/test_support"
+    sh "git -C test_support checkout v2.0.0"
+  }
+}
```

Comment:
Ideally clone from your tests/requirements.txt

It makes it easier to setup and run tests locally and also feels nicer than clones in jenkinsfile

I recently made a similar change to lib_xua here: https://github.com/xmos/lib_xua/pull/516/files